src/index/updater.rs

```diff
@@ -229,7 +229,7 @@ impl Updater {
 
           errors += 1;
           let seconds = 1 << errors;
-          log::error!("failed to fetch block {height}, retrying in {seconds}s: {err}");
+          log::warning!("failed to fetch block {height}, retrying in {seconds}s: {err}");
```

Comment:
I think this should be `log::warn`

Reply:
Perfect, fixed. Assume it's good to go otherwise? Are there any recoverable errors that should be warnings?